src/styles/LocationDetails.css

```diff
@@ -0,0 +1,6 @@
+@import url("https://fonts.googleapis.com/css2?family=Nunito:wght@300&family=Open+Sans:wght@300&family=Poppins:wght@200&display=swap");
```

Comment:
As above you arent using the nunito font so that part of the import can be removed